crates/pessimistic-proof-test-suite/tests/snapshots/vkey_selector__vkey_snapshot.snap

```diff
@@ -1,8 +1,9 @@
 ---
 source: crates/pessimistic-proof-test-suite/tests/vkey_selector.rs
 expression: message
+snapshot_kind: text
 ---
 # If this test fails, it means the PP vkey has changed.
 # When that happens, consider updating the selector by bumping the PP version.
-| PP_VKEY          | 0x005501f27f2c36c1540e97f3f2997abd8830cffefca4a5e2759a5604a763e748 |
+| PP_VKEY          | 0x006d77897e2eb9f59938982739f46f93febf1c04aaa7d4f1539d1bc02785afc8 |
```

Comment:
Do we want to bump the selector here or was it already done for this release/iteration? @hadjiszs